examples/policies/ngx-example/1.0.0/init.lua

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
I think we should follow the same pattern we follow for our policies, that is, having a `init.lua` with only a `require` and the code in a different .lua file.

Reply:
We should probably add a json schema as well to show a complete example.

Reply:
Yep!

Reply:
First part done. Will do the schema now.

Reply:
Done in https://github.com/3scale/apicast/pull/581/commits/e6785ae55b818ec83b0a2f273b4a2b5fbe89a88e